### DISCUSSION TITLE
suggestion

### DISCUSSION THREAD

**Discussion by SrMonsterYT** [BG-Software-LLC/SuperiorSkyblock2#1834] [Approved Feature Requests]
in the Askyblock plugin of 1.12.2 there was an option that when you plant an oak tree in the nether the gravel trunk and the glowstone tree were born.

could you add this option that when you plant an oak tree in the nether the gravel trunk and glowstone leaves come?

**Comment by Tuogptuan** [comment]
Prob u should find a plugin for this
